renderer/pages/index.tsx

```diff
@@ -617,7 +617,7 @@ const Home = () => {
               <img
                 src={
                   "file://" +
-                  `${upscaledImagePath ? upscaledImagePath : imagePath}`
+                  `${upscaledImagePath ? upscaledImagePath.replace(/([^/\\]+)$/i, encodeURIComponent(upscaledImagePath.match(/[^/\\]+$/i)[0])) : imagePath.replace(/([^/\\]+)$/i, encodeURIComponent(imagePath.match(/[^/\\]+$/i)[0]))}`
```

Comment:
I think it'd be good to modularize this snippet. Put the separate values in consts or functions and call those instead of having less readable code here.

Some documentation on why we're using all the regex would be great as well!

Reply:
I have made the changes, please check when you can